Examples/Python/python/acts/__init__.py

```diff
@@ -1,12 +1,31 @@
 from pathlib import Path
 from typing import Union
+import os
+import warnings
 
 
 from .ActsPythonBindings import *
 from .ActsPythonBindings import __version__
 from . import ActsPythonBindings
 from ._adapter import _patch_config
 
+if (
+    "ACTS_LOG_FAILURE_THRESHOLD" in os.environ
+    and os.environ["ACTS_LOG_FAILURE_THRESHOLD"] != logging.getFailureThreshold().name
+):
+    error = (
+        "Runtime log failure threshold is given in environment variable "
+        f"`ACTS_LOG_FAILURE_THRESHOLD={os.environ['ACTS_LOG_FAILURE_THRESHOLD']}`"
+        "However, a compile-time value is set via CMake, i.e. "
+        f"`ACTS_LOG_FAILURE_THRESHOLD={logging.getFailureThreshold().name}`. "
+        "or `ACTS_ENABLE_LOG_FAILURE_THRESHOLD=OFF`, which disables runtime thresholds."
```

Comment:
Can you do the same edit on the message of the std::runtime_error emitted on the C++ side?